src/server/selection/epsilonGreedySelection.test.ts

```diff
@@ -66,7 +66,11 @@ describe('selectVariantWithHighestMean', () => {
             testName: 'example-1',
             sortedVariants: [{ variantName: 'v1', mean: 1 }],
         };
-        expect(selectVariantWithHighestMean(banditData, epicTest)?.name).toEqual('v1');
```

Comment:
I've updated this test case as well. It's supposed to test the case where there's only one variant